pkg/apis/core/validation/validation.go

```diff
@@ -5593,6 +5594,37 @@ func validatePodResourceClaimStatuses(statuses []core.PodResourceClaimStatus, po
 	return allErrs
 }
 
+// validatePodExtendedResourceClaimStatus validates the ExtendedResourceClaimStatus in a pod status.
+func validatePodExtendedResourceClaimStatus(status *core.PodExtendedResourceClaimStatus, containers []core.Container, fldPath *field.Path) field.ErrorList {
+	if status == nil {
+		return nil
+	}
+	var allErrs field.ErrorList
+
+	for i, rm := range status.RequestMapping {
+		idxPath := fldPath.Index(i)
```

Comment:
```suggestion
		idxPath := fldPath.Child("requestMapping").Index(i)
```

This could be pulled out of the loop, but then you need another `fldPath` variable because below you are validating something which is not in the `requestMapping`.